codegen_glibmm/templates/stub.h.templ

```diff
@@ -55,8 +55,27 @@ protected:
 {% for signal in interface.signals %}
 {% if signal is supported_by_sigc %}
 
-    void {{ signal.name }}_emitter({{ signal.args|map(attribute='cpptype_out')|join(', ') }});
+    void {{ signal.name }}_emitter(const std::vector<Glib::ustring> &destination_bus_names, {{ signal.args|map(attribute='cpptype_out')|join(', ') }}&);
+    inline void {{ signal.name}}_multicastEmitter(const std::vector<Glib::ustring> &destination_bus_names,
+    {%- set comma = joiner() -%}
+    {%- for arg in signal.args -%}
+    {{ comma() }}{{ arg.cpptype_out }} {{ arg.name }}
+    {%- endfor -%}) {
+        {% for arg in signal.args %}
+        {{ signal.name }}_emitter(destination_bus_names, {{ arg.cpptype_to_dbus }}{{ arg.name }});
```

Comment:
I found a bug that multiple emitter function is generated when signal has a multiple argument.
 it will be change below like
  {{ signal.name }}_emitter(destination_bus_names,
                 {%- set comma = joiner() -%}
                 {%- for arg in signal.args -%}
                {{ comma() }}{{ arg.name }}
               {%- endfor -%});

Reply:
@mardy 
I was fixed that bug 
Please review again.